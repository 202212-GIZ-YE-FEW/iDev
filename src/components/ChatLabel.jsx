function ChatLabel(props) {
    const { label } = props;

    return (
        <div className='mt-10 flex items-center text-center text-sm text-white mx-auto bg-gray/40 shadow-sm rounded-full px-4 py-1 border border-gray/10'>
            {label}
        </div>
    );
}
export default ChatLabel;
