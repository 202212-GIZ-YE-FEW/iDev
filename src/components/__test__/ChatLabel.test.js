import renderer from "react-test-renderer";

import ChatLabel from "../ChatLabel";

it("renders correctly", () => {
    const tree = renderer.create(<ChatLabel />).toJSON();
    expect(tree).toMatchSnapshot();
});
