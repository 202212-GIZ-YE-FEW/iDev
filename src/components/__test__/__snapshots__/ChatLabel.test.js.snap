// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="mt-10 flex items-center text-center text-sm text-white mx-auto bg-gray/40 shadow-sm rounded-full px-4 py-1 border border-gray/10"
/>
`;
